oozie-to-airflow/mappers/email_mapper.py

```diff
@@ -12,14 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import os
-import subprocess
-
 import jinja2
+
+from airflow.operators import email_operator
 from airflow.utils.trigger_rule import TriggerRule
 
-from definintions import ROOT_DIR
```

Comment:
This should be fixed in the prior commit. Each commit should be good to go stand-alone.